tests/unittests/BackendCorrectnessTest.cpp

```diff
@@ -597,6 +597,18 @@ TEST_P(CPUOnly, groupConvTest) {
   EXPECT_TRUE(out1.isEqual(out2));
 }
 
+/// This test targets the DKKC8 optimization.
+TEST_P(CPUOnly, nonSquarePaddingConvTest) {
+  std::array<size_t, 4> S{{1, 2, 1, 128}};
+  llvm::ArrayRef<size_t> shape(S);
+  Tensor out1(ElemKind::FloatTy, shape);
+  Tensor out2(ElemKind::FloatTy, shape);
+  inferNonSquarePaddingConv(&out1, backendKind_);
```

Comment:
backendKind_ -> BackendKind::CPU ?

Reply:
I understand. But BackendKind::CPU is more readable in my opinion. Unprepared reader may have a question, why do we even iterate on backendKind_